cmd/svcat/instance/provision_cmd.go

```diff
@@ -45,6 +46,7 @@ func NewProvisionCmd(cxt *command.Context) *cobra.Command {
 		Short: "Create a new instance of a service",
 		Example: `
   svcat provision wordpress-mysql-instance --class mysqldb --plan free -p location=eastus -p sslEnforcement=disabled
+  svcat provision wordpress-mysql-instance --external-id wordpress-mysql-instance-external-id --class mysqldb --plan free
```

Comment:
Same as before, I'd like to see a representative value used for the external id here.